### ISSUE TITLE
/delkit command is case sensitive, even though /kit isn't

### ISSUE THREAD

**Issue by ToonTown0909** [EssentialsX/Essentials#3370]
### Information

**Full output of `/ess version`**:

```
12.06 04:40:36 [Server] INFO CONSOLE issued server command: /ess version
12.06 04:40:36 [Server] INFO Server version: 1.15.2-R0.1-SNAPSHOT git-Paper-334 (MC: 1.15.2)
12.06 04:40:36 [Server] INFO EssentialsX version: 2.17.2.116
12.06 04:40:36 [Server] INFO PlaceholderAPI version: 2.10.6
12.06 04:40:36 [Server] INFO PermissionsEx version: 1.23.4
12.06 04:40:36 [Server] INFO Vault version: 1.7.2-b107
12.06 04:40:36 [Server] INFO Citizens version: 2.0.26-SNAPSHOT (build 1982)
12.06 04:40:36 [Server] INFO EssentialsXChat version: 2.17.2.116
12.06 04:40:36 [Server] INFO ChestShop version: 3.10-SNAPSHOT (build 151)
12.06 04:40:36 [Server] INFO You are running unsupported plugins!
```

**Server log**: https://gist.github.com/ToonTown0909/729f133dfdf53175dd699584d91c21e3

**EssentialsX config**: https://gist.github.com/ToonTown0909/bfc3e2ba2cc80d7f3495454e1a9fb43a

### Details

**Description**  
The /delkit command fails to work consistently. 

**Steps to reproduce**  
Use the /delkit command.

**Expected behavior**  
The named kit is expected to no cease its existance.

**Screenshots**  
<img width="987" alt="Screen Shot 2020-06-11 at 10 51 12 PM" src="https://user-images.githubusercontent.com/2454717/84460027-d598e500-ac36-11ea-97aa-97474ad7e456.png">
<img width="985" alt="Screen Shot 2020-06-11 at 10 51 27 PM" src="https://user-images.githubusercontent.com/2454717/84460037-dcbff300-ac36-11ea-97cf-7777be3e2800.png">

**Comment by pop4959** [comment]
Doesn't appear that the command is broken, but what I noticed looking into this is that the input for /delkit is case sensitive. That means if you created the kit and named it "Kit", you have to delete it as "Kit" (not "kit", etc). You should be able to see what it was named in kits.yml.

**Comment by ToonTown0909** [comment]
Am rather embarrassed I didn't figure this out on my own. Very grateful for the help! Thank you!

**Comment by pop4959** [comment]
It's still a problem because, at the very least, the tab completions should not be lower cased when trying to delete kits that contain capital letters. Also, I don't think kits are supposed to be case sensitive at all anyways (since they are formatted by Essentials when listing them), so it might also make sense to normalize the name when creating them. Let's leave this open until someone can look into it and put in some kind of fix so that this won't confuse anyone else.

**Comment by ToonTown0909** [comment]
I think that'd be great, thank you every much. Is there anything I can do on my end to help things along? Either way, very grateful. :)

**Comment by pop4959** [comment]
@ToonTown0909 If you're a programmer, you could put in a pull request yourself with an appropriate fix. Otherwise you can just relax for now, and someone else might decide to take a look at it.

**Comment by triagonal** [comment]
This has been fixed in EssentialsX versions `2.17.2.142` and above. Download the latest build [here](https://essentialsx.net/downloads.html).
